code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -198,27 +198,28 @@
 
 /obj/item/robot_module/proc/do_transform_animation()
 	var/mob/living/silicon/robot/R = loc
-	R.notransform = TRUE
-	var/obj/effect/temp_visual/decoy/fading/fivesecond/ANM = new /obj/effect/temp_visual/decoy/fading/fivesecond(R.loc, R)
-	ANM.layer = R.layer - 0.01
-	new /obj/effect/temp_visual/small_smoke(R.loc)
 	if(R.hat)
 		R.hat.forceMove(get_turf(R))
 		R.hat = null
-	R.update_headlamp()
-	R.alpha = 0
-	animate(R, alpha = 255, time = 50)

```

Comment:
borgs without transform animations will now lock in place and suddenly swap to the new sprite.

Reply:
an acceptable loss. Though putting animations in reverse as if 'repacking' the module -> standard would work.

Reply:
It would be better if you kept the current behavior for when an animation is unavailable.

Reply:
That's how it was before, but then it was ask to not do copy/paste overrides for all borgs, so here we are.